tests/e2e/users/add-user.specs.js

```diff
@@ -1,24 +1,41 @@
-const utils = require('../../utils');
-const usersPage = require('../../page-objects/users/users.po.js');
-const helper = require('../../helper');
-const addUserModal = require('../../page-objects/users/add-user-modal.po.js');
-
-const addedUser = 'fulltester' + new Date().getTime();
-const fullName = 'Bede Ngaruko';
-const errorMessagePassword = element(by.css('#edit-password ~ .help-block'));
+const utils=require('../../utils');
```

Comment:
Are all the spaces being stripped out on purpose? Makes it a little hard to see what actually changed.

Reply:
Used a different machine with different text editor's settings - will fix that